stubs/ratelimit/ratelimit/decorators.pyi

```diff
@@ -0,0 +1,16 @@
+from collections.abc import Callable
+from typing import TypeVar
+from typing_extensions import ParamSpec
+
+_P = ParamSpec("_P")
+_T = TypeVar("_T")
+
+now: Callable[[], float] = ...
```

Comment:
I don't think this should be included in the stub. The re-export seems incidental. On the other hand it could make sense to include the (very simple) `ratelimit.utils` module just to complete the stubs.

Reply:
I removed `now`.

Reply:
> I can include the utils module but that made it fail the check because only the latest HEAD has it but not what is on PyPI.

Ah, I didn't notice that.

Reply:
In that case it's probably best to add

```
ratelimit.decorators.now
```

to `stubs/ratelimit/@tests/stubtest_allowlist.txt`.